Gitpod (#169)
palantirnet/drupal8-rector: README.md

Our goal is to make contributing to this project easy for people. While we've made certain architectural decisions here to hopefully achieve that goal, it's a work in progress and feedback is appreciated.

## Development environment

> We recommend using Gitpod, a full development environment in the cloud.

Reviewer: I don't think this language is agreed upon. "You may use Gitpod" is likely better.

Author: ```suggestion
You may use Gitpod, a full development environment in the cloud.
```
